app/routes/device.py

```diff
@@ -70,11 +70,24 @@ def update_device(deviceId):
     updated_row = Database.update_device(deviceId, **body)
 
     if updated_row is None:
+        Database.write_log(
+            role      = g.tokenPayload["role"],
+            action    = f"User {g.tokenPayload['user_id']} tried updating unknown device {deviceId}",
+            user_id   = g.tokenPayload["user_id"],
+            device_id = deviceId,
+        )
         return errorResponse(
         error = "Device not found",
         statusCode = 404
     )
     else:
+        Database.write_log(
+            role      = g.tokenPayload["role"],
+            action    = f"User {g.tokenPayload['user_id']} updated device {deviceId}",
+            user_id   = g.tokenPayload["user_id"],
+            device_id = deviceId,
+        )
```

Comment:
ples fix indentations
```suggestion
    if updated_row is None:
        Database.write_log(
            role      = g.tokenPayload["role"],
            action    = f"User {g.tokenPayload['user_id']} tried updating unknown device {deviceId}",
            user_id   = g.tokenPayload["user_id"],
            device_id = deviceId,
        )

        return errorResponse(
            error = "Device not found",
            statusCode = 404
        )

    Database.write_log(
        role      = g.tokenPayload["role"],
        action    = f"User {g.tokenPayload['user_id']} updated device {deviceId}",
        user_id   = g.tokenPayload["user_id"],
        device_id = deviceId,
    )
```